File: Strategy Summit/Game Theory/Q1. Nim Game/Q1_NimGame.java
class Solution {
    public boolean canWinNim(int n) {
        // You lose only when n is a multiple of 4
        // because whatever you pick (1, 2, or 3),
        // the opponent can always pick to make the sum 4
        return n % 4 != 0;
    }
}

File: Strategy Summit/Game Theory/Q1. Nim Game/Q1_NimGame.py
class Solution(object):
    def canWinNim(self, n):
        """
        :type n: int
        :rtype: bool
        """
        # You lose only when n is a multiple of 4
        # because your opponent can always take enough stones
        # to make the total removed each round equal to 4
        return n % 4 != 0
